demo/scripts/script-test.js

```diff
@@ -1,4 +1,5 @@
 var notyf;
+var notyfNotifications = [];
```

Comment:
Currently maintaining the opened notification here so that they can be accessed easily to test dismiss functionality for them. Not the most elegant solution, so I'm not very sure of this.
Another way could be listing all the active notifications separately, and showing a dismiss button beside them, but clearly, that would require quite a lot of work.
What are your thoughts about this? What could be done instead? @caroso1222